src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator {
+
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    @Override
+    public String toString() {
+        return "Figure: ";
+    }
```

Comment:
why do we need this?

Reply:
Hello. According to the task, figures should be printed out with the word "Figure " before. Please check the task description.   Or I can add this word to each class toString method.

Reply:
![image](https://user-images.githubusercontent.com/20326858/210266285-4cd72d87-c8ed-4359-976f-181675d3daad.png)